src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx

```diff
@@ -0,0 +1,160 @@
+    id="step3"
+    title="Step 3. Update `config.yaml`"
+    >
+To ensure that you're using the most up to date information for the integration, update your `config.yaml` file to match the snippet found in [Key Pair authentication](https://github.com/newrelic/newrelic-snowflake-integration#key-pair-authentication) in the Snowflake repo. 
+
+Once you've updated the yaml file, return to this doc to install the dashboards to your New Relic account. 
+    </Collapser>
+</CollapserGroup>
+
+## Create custom queries [#custom]
+
+`TEXT HERE PENDING PHANEE'S RESPONSE` 
+
+<CollapserGroup>
+    <Collapser
+        className="freq-link"
+        id="longest"
+        title="Longest queries"
+    >
+    To find the queries that take the longest to load:
+
+    1. Create a file and name it `longest_queries.sql` 
+    2. Designate the file path as `newrelic-snowflake-integrations/queries`.
+    3. In `longest_queries.sql`, add the following snippet:
+
+    ```sql
+    select 
+        query_id, 
+        query_text,
+        (execution_time / 60000) as exec_time,
+        warehouse_name,
+        user_name,
+        execution_status
+    from 
+        snowflake.account_usage.query_history 
+    where 
+        execution_status = 'SUCCESS' 
+    order by 
+        execution_time desc
+    ```
+    4. In the `flex-snowflake.yml` file, add the following snippet:
+    
+```yml
+- name: longestQueries
+entity: snowflake
+event_type: SnowflakeVirtualWarehouse
+custom_attributes:
+    metric_type: snowflake.query_performance
+commands: 
+    - run: <var>YOUR_PATH_TO snowflakeintegration-platform</var> <var>YOUR_PATH_TO config.yaml</var> <var>YOUR_PATH_TO longest_queries.sql</var>
+```
+    </Collapser>
+    <Collapser
+        className="freq-link"
+        id="warehouse-usage"
+        title="Usage by warehouse"
+    >
+    To find the queries that take the longest to load:
+
+    1. Create a file and name it `credit_usage_by_warehouse.sql` 
+    2. Designate the file path as `newrelic-snowflake-integrations/queries`.
+    3. In `longest_queries.sql`, add the following snippet:
+
+    ```sql
+    select 
+        warehouse_name, 
+        sum(credits_used) as total_credits_used 
+    from 
+        snowflake.account_usage.warehouse_metering_history 
+    group by 
+        1 
+    order by 
+        2 desc;
+    ```
+    4. Add the following snippet to `flex-snowflake.yml`:
+
+    ```yml
+    - name: CreditUsageByWarehouse
+        entity: snowflake
+        event_type: SnowflakeVirtualWarehouse
+        custom_attributes:
+            metric_type: snowflake.credit_usage_by_warehouse
+        commands: 
+        - run: <var>YOUR_PATH_TO snowflakeintegration-platform</var> <var>YOUR_PATH_TO config.yaml</var> <var>YOUR_PATH_TO credit_usage_by_warehouse.sql</var>
+    ``` 
+    </Collapser>
+</CollapserGroup>
+
+## Get Snowflake metrics in a dashboard [#dash]
+
+Once you've installed the Snowflake quickstart, you can see your critical Snowflake data in New Relic. 
```

Comment:
Once you've installed the [Snowflake quickstart](https://newrelic.com/instant-observability/snowflake), you can see your critical Snowflake data in New Relic.